easybuild/easyconfigs/p/Python/Python-3.6.4-fosscuda-2018a.eb

```diff
@@ -0,0 +1,171 @@
+name = 'Python'
+version = '3.6.4'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'fosscuda', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['7dc453e1a93c083388eb1a23a256862407f8234a96dc4fae0fc7682020227486']
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.11'),
+    ('libreadline', '7.0'),
+    ('ncurses', '6.0'),
+    ('SQLite', '3.21.0'),
+    ('GMP', '6.1.2'),  # required for pycrypto
+    ('libffi', '3.2.1'),  # required for cryptography
```

Comment:
@akesandgren missing `XZ` dependency...

Reply:
Odd, it's there in the .eb file I'm using to actually build...